README.md

```diff
@@ -24,42 +24,8 @@ The figure below illustrates the software flow through the architecture componen
 5. **Publish Inference Events/Data:** Analytics inferences are formatted and passed to the destination message bus specified in the request.
 6.  **Export Data:** Publish prepared (transformed, enriched, filtered, etc.) and groomed (formatted, compressed, encrypted, etc.) data to external systems (be it analytics package, enterprise or on-premises application, cloud systems like Azure IoT, AWS IoT, or Google IoT Core, etc.
 
-
-# Getting Started
-
-For a full walkthrough of using the default images, use this [guide.](./doc/guides/SimpleStartupGuide.md)  
-
-For a full walkthrough of building custom images, use this [guide.](./doc/guides/CustomStartupGuide.md)  
-
-A brief video demonstration of building and using the device service can be found [here.](https://www.youtube.com/watch?v=vZqd3j2Zn2Y)
-
 # Learn More 
-
-### General
-[Supported ONVIF features](./doc/ONVIF-protocol.md)  
-[Auto discovery](./doc/auto-discovery.md)  
-[Control-plane events](./doc/control-plane-events.md)  
-[Utility Scripts](./doc/utility-scripts.md)
-
-
-### Custom Features
-[Custom Metadata](./doc/custom-metadata-feature.md)  
-[Reboot Needed](./doc/custom-feature-rebootneeded.md)  
-[Friendly Name and Mac Address](./doc/get-set-friendlyname-mac.md)  
-
-### API Support
-[API Analytic Handling](./doc/api-analytic-support.md)  
-[API Event Handling](./doc/api-event-handling.md)  
-[API User Handling](./doc/api-usage-user-handling.md)  
-
-### Miscellaneous
-[Postman](./doc/test-with-postman.md)  
-[User Authentication](./doc/onvif-user-authentication.md)  
-
-## Resources
-[Learn more about EdgeX Core Metadata](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.1.0)  
-[Learn more about EdgeX Core Command](https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-command/2.1.0)
-
+Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/2.3/microservices/device/supported/device-onvif-camera/general) for more details.
```

Comment:
The docs have a special version called `latest` which will always navigate to that latest release.

```suggestion
Visit the [EdgeX documentation page](https://docs.edgexfoundry.org/latest/microservices/device/supported/device-onvif-camera/general) for more details.
```